packages/frames.js/src/constants.ts

```diff
@@ -1,2 +1,4 @@
 export const FRAMESJS_DEBUG_INFO_IMAGE_KEY =
   "frames.js:debug-info:image" as const;
+
+export const DEFAULT_FRAME_TITLE = "frame";
```

Comment:
perhaps we can update this to "Frames.js Frame"? is it used in the examples?

Reply:
we need to update the templates page.tsx generateMetadata functions to use this as well so we can detect and warn when it needs to be updated

Reply:
Nope, this is not used in examples. This is used only if you use `getFrameHtml()`, that is not used in `fetchMetadata()` since that uses `getFrameFlattened()`.

Reply:
examples have their own titles

Reply:
changed to `Frames.js Frame`, at least it will force people to change it.

Reply:
hm i would say we should be able to detect if the base template title hasn't been changed yet

we can omit the title property from `generateMetadata` and then it would use the default one, but it might not be obvious where to change the title...

perhaps we should add a title/description option to createFrames? what do you think?

Reply:
the thing is that `generateMetadata()` doesn't use `title` at all. So that would require a little bigger refactor since we would need to extend some types and functions to actually support this. Also should it generate `title` tag or `og:title` instead?

Reply:
Also `fetchMetadata()` result is passed to `other` in `generateMetadata()` so I guess we can generate only `og:title` (which is fine because `og:title` takes precedence over `title`.

Reply:
we have the option to ship this with a clear error message about how to change the title and maybe some extra docs. then we can push the bigger refactor to another pr

Reply:
@stephancill I don't follow. How should that warning logic work?

Reply:
the warning is fine, perhaps we don't need explicit instructions about how to change the title

Reply:
Do you mean warning that is shown in debugger? Or do you want some special warning when using fetchMetadata() ? That you should specify og:title or title?

Reply:
i'm referring to what we have in the debugger via getFrame at the moment

Reply:
Ok so in that case it is already done right?

Reply:
yes

Reply:
Ok so good to merge? 😀

Reply:
yessir